.ruby-version

```diff
@@ -0,0 +1 @@
+ruby-1.9.3
```

Comment:
I think that we don't need to force all developers to use 1.9.3. We can use any ruby version to develop because the travis is checking all commits with all ruby versions that we support.
It's basic the same think that I think about .ruby-gemset.

WDYT?